File: ChristmasTreeApp/lib/models/stylistbooking.dart
class StylistBooking {
  final int stylistId;
  final String stylistName;
  final DateTime bookingDate;
  final int addressId;
  final String? notes;

  StylistBooking({
    required this.stylistId,
    required this.stylistName,
    required this.bookingDate,
    required this.addressId,
    this.notes,
  });

  factory StylistBooking.fromJson(Map<String, dynamic> json) {
    return StylistBooking(
      stylistId: json['stylistId'],
      stylistName: json['stylistName'],
      bookingDate: DateTime.parse(json['bookingDate']),
      addressId: json['addressId'],
      notes: json['notes'],
    );
  }

  Map<String, dynamic> toJson() {
  return {
    'stylistId': stylistId,
    'stylistName': stylistName,
    'bookingDate': bookingDate.toIso8601String(),
    'addressId': addressId,
    'notes': notes,
  };
}

}

File: ChristmasTree_API/Models/StylistBooking.cs
namespace ChristmasTreeApp.Models
{
    public class StylistBooking
    {
        public int StylistBookingId { get; set; }
        public int OrderId { get; set; }
        //public Order Order { get; set; }
        public int StylistId { get; set; }
        //public TreeStylist Stylist { get; set; }
        public DateTime BookingDate { get; set; }
        public int AddressId { get; set; }
        //public Address Address { get; set; }
        public string Notes { get; set; }
        public string Status { get; set; }
    }
}
